tinc.yaml

```diff
@@ -0,0 +1,3 @@
+dependencies:
+  - name: riak-protobuf
+    path: protobuf
```

Comment:
> is that tinc doesn't support caching of source dependencies
I think it does if it's specified by git repo & commit.

@jystic I'll take a look at mafia